modules/cats-effect/src/main/scala/io/janstenpickle/trace4cats/cats/effect/LocalForIOLocal.scala

```diff
@@ -0,0 +1,26 @@
+package io.janstenpickle.trace4cats.cats.effect
+
+import cats.Monad
+import cats.effect.{IO, IOLocal, Resource}
+import io.janstenpickle.trace4cats.Span
+import io.janstenpickle.trace4cats.base.context.Local
+import io.janstenpickle.trace4cats.base.optics.Lens
+
+private[trace4cats] class SpanLocalForIOLocal[Ctx](rootSpan: IOLocal[Ctx])(implicit lens: Lens[Ctx, Span[IO]])
```

Comment:
This implementation can be gotten via `ContextLocalForIOLocal(rootSpan).focus(lens)`.

Reply:
@ybasket could you weigh on in this one? It's not immediately obvious to me whether this would work for the few cases that are currently using `SpanLocalForIOLocal`

Reply:
Yes, it should work. I initially tried to get away with one class and the right `Lens` and failed, but the types align with `focus` and a quick test seemed fine (I also don't see it could go wrong).

Commit: https://github.com/ybasket/trace4cats/commit/7b0ede36a40a3dd0099b163b3cff2830dd388ad9

Reply:
Thanks @ybasket, just pulled in that commit as well -- @catostrophe when you have a chance could you double check that this looks like you'd hoped now?